docker/test-images/zipkin-cassandra/install.sh

```diff
@@ -133,45 +113,35 @@ log4j.appender.stdout.Target=System.out
 EOF
 
 cat zipkin-schemas/zipkin2-schema.cql zipkin-schemas/zipkin2-schema-indexes.cql > schema
-case ${CASSANDRA_VERSION} in
-  3.11* )
-    cqlversion=3.4.4
-    ;;
-  4* )
-    # read_repair_chance options were removed and make Cassandra crash starting in v4
-    # See https://cassandra.apache.org/doc/latest/operating/read_repair.html#background-read-repair
-    sed -i '/read_repair_chance/d' schema
-    cqlversion=3.4.5
-    ;;
-esac
-
-jdk11_modules="--add-exports java.base/jdk.internal.misc=ALL-UNNAMED \
+
+# read_repair_chance options were removed and make Cassandra crash starting in v4
+# See https://cassandra.apache.org/doc/latest/operating/read_repair.html#background-read-repair
+sed -i '/read_repair_chance/d' schema
+
+# Run cassandra on a different port temporarily in order to setup the schema.
+# We also add exports and opens from Cassandra 4, except RMI, which isn't in our JRE image.
+# See https://github.com/apache/cassandra/blob/cassandra-4.0.11/conf/jvm11-server.options
+java -cp 'classes:lib/*' -Xms64m -Xmx64m -XX:+ExitOnOutOfMemoryError -verbose:gc \
+  -Djdk.attach.allowAttachSelf=true \
+  --add-exports java.base/jdk.internal.misc=ALL-UNNAMED \
   --add-exports java.base/jdk.internal.ref=ALL-UNNAMED \
   --add-exports java.base/sun.nio.ch=ALL-UNNAMED \
   --add-exports java.sql/java.sql=ALL-UNNAMED \
+  --add-opens java.base/java.lang=ALL-UNNAMED \
```

Comment:
This one seems new though util/concurrent/concurrent.atomic aren't so maybe it's ok, but I was surprised since they seem to open packages that are already public.

I don't see them in the linked jvm11-server.options, if they're needed, is there a way to document how to find out what to write here?

Reply:
Added a commit that may make sense. Main idea is to keep hunting down since this is a test image. Merge in anything cassandra 5 needs for JRE 17+, so that we can use the same base for cassandra 4 without a lot of sleuthing.